cmd/kubeadm/app/cmd/phases/reset/unmount_linux.go

```diff
@@ -32,6 +32,11 @@ func unmountKubeletDirectory(absoluteKubeletRunDirectory string) error {
 	if err != nil {
 		return err
 	}
+
```

Comment:
@thtanaka can we add a comment above the `if` statement as to why this is necessary for posterity?